mdoc/src/main/scala/mdoc/internal/worksheets/WorksheetProvider.scala

```diff
@@ -75,21 +73,21 @@ class WorksheetProvider(settings: Settings) {
       settings.screenWidth - statement.position.endColumn
     )
     val isEmptyValue = isUnitType(statement) || statement.binders.isEmpty
-    val contentText = renderContentText(statement, margin, isEmptyValue)
-    val hoverMessage = renderHoverMessage(statement, margin, isEmptyValue)
-    EvaluatedWorksheetStatement(range, contentText, hoverMessage)
+    val RenderSummaryResult(summary, isSummaryComplete) =
```

Comment:
Let’s not use val destructuring. It’s better to  bind a normal variable and access fields

Reply:
done